metrics/metrics.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"time"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"k8s.io/component-base/metrics"
+)
+
+const (
+	// Common metric strings
+	subsystem             = "csi_sidecar"
+	labelCSIDriverName    = "driver_name"
+	labelCSIOperationName = "method_name"
+	labelGrpcStatusCode   = "grpc_status_code"
+	unknownCSIDriverName  = "unknown-driver"
+
+	// CSI Operation Total with status code - Count Metric
```

Comment:
Do we still need this? xref: https://github.com/kubernetes-csi/external-provisioner/pull/386#discussion_r359129206

Reply:
Ack, will remove.